modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -1305,9 +1313,9 @@ export default class Ecdsa {
    * @param {DShare} dShare delta share received from the other participant
    * @param {Hash} hash hashing algorithm implementing Node`s standard crypto hash interface
    * @param {boolean} shouldHash if true, we hash the provided buffer before signing
-   * @returns {SShare}
+   * @returns {VAShare}
    */
-  sign(M: Buffer, oShare: OShare, dShare: DShare, hash?: Hash, shouldHash = true): SShare {
+  sign(M: Buffer, oShare: OShare, dShare: DShare, hash?: Hash, shouldHash = true): VAShare {
```

Comment:
Will this break the existing signing flow for clients or anyone using the latest sdk?

Reply:
As far as I know, it shouldn't as `VAShare` is subtype of `SShare`.  In other words, everything from `SShare` is there and remains the same.